accounts/src/main/java/eu/kevin/accounts/bankselection/BankSelectionViewModel.kt

```diff
@@ -198,13 +205,20 @@ internal class BankSelectionViewModel constructor(
     }
 
     @Suppress("UNCHECKED_CAST")
-    class Factory(owner: SavedStateRegistryOwner) : AbstractSavedStateViewModelFactory(owner, null) {
+    class Factory(
+        private val context: Context,
+        owner: SavedStateRegistryOwner
+    ) : AbstractSavedStateViewModelFactory(owner, null) {
         override fun <T : ViewModel?> create(
             key: String,
             modelClass: Class<T>,
             handle: SavedStateHandle
         ): T {
             return BankSelectionViewModel(
+                DefaultCountryIsoProvider(
+                    context,
```

Comment:
Small nitpick but we could avoid passing context here and something depending on context (TelephonyManager) by just providing this class what it needs:

```kotlin
DefaultCountryIsoProvider(
    currentLocale = context.getCurrentLocale(),
    packageManager = context.packageManager,
    telephonyManager = context.getSystemService(Context.TELEPHONY_SERVICE) as TelephonyManager,
),
```

Reply:
Generally, I agree with you. But technically that way there is a micro possibility that while user is on BankSelection screen, he could go to system settings and change locale, then DefaultCountryIsoProvider will have outdated locale.